alerts/plugins/possible_usernames.json

```diff
@@ -0,0 +1,4 @@
+{
+  "search_window_hours": 24,
+  "indices_to_search": ["events-*"]
```

Comment:
It might be more efficient to just search on `events` and `events-previous`, although that assumes a search window of 24 hours max.

Reply:
Given that this is a configuration file, these values are intended to be easy to change.  Want to leave these as the defaults if for no other reason than because it's consistent with other defaults?